installation/post-installation-guide.md

```diff
@@ -90,6 +120,14 @@ The following configuration uses the `in_forward` plugin as an input source and
 
 Fluentd manages plugins as Ruby gems but stores them in their dedicated directory separated from the normal Ruby gems.
 
+#### `fluent-package`
+
+A special program `fluent-gem` is used to manage plugin gems. For example, the following command installs a plugin to connect to S3 \(including both `in_s3` and `out_s3` plugins\):
+
+```text
+ $ sudo /usr/sbin/fluent-gem install fluent-plugin-s3
```

Comment:
```suggestion
 $ sudo fluent-gem install fluent-plugin-s3
```

Would a full path be better?
I know the existing other descriptions use the full path. 
If unifying them would be better, that's fine.